src/Webform.js

```diff
@@ -1526,8 +1526,11 @@ export default class Webform extends NestedDataComponent {
     this.submitting = true;
     return this.submitForm(options)
       .then(({ submission, saved }) => this.onSubmit(submission, saved))
-      .catch((err) => NativePromise.reject(this.onSubmissionError(err)))
-      .finally(() => this.submissionInProcess = false);
+      .then(() => this.submissionInProcess = false)
```

Comment:
Won't this change the response from the promise?  Instead of returning the results of "onSubmit" it will now return "false".  I am sure this will break the renderer.